cluster/gce/gci/mounter/mounter

```diff
@@ -48,8 +48,10 @@ echo "Running mount using a rkt fly container"
 
 ${RKT_BINARY} run --stage1-path=${STAGE1_ACI} \
 	--insecure-options=image \
-	--volume=kubelet,kind=host,source=/var/lib/kubelet,readOnly=false,recursive=true \
-	--mount volume=kubelet,target=/var/lib/kubelet \
+	--volume=var,kind=host,source=/var,readOnly=false,recursive=true \
+	--mount volume=var,target=/var \
```

Comment:
FYi: This might result in a lot of mounts showing up inside rkt pods. I'd recommend a couple of days of testing to ensure we don't hit mount related issues you had previously identified..